### ISSUE TITLE
Write an email service to check a simple IMAPS server, for a new token

### ISSUE THREAD

**Issue by nicksauca** [tritus/candilibre#7]

**Comment by tritus** [comment]
Could you provide more details? I don't want the user to have to specify any email authentication. That would be too techy for them in my opinion.
Furthermore, I'm quite bad at email stuff. Does IMAPS provide easy authentication?

**Comment by nicksauca** [comment]
Well, you see, i created two issues cuz there are two steps: 1. ask for a new token (this step is done by calling candilib api) 2. retrieve this token from users email account.

these two steps can be implemented separately

For step one, you will only need users email adress (no email authentication)
The stem two is more tricky: use of IMAPS servers, the authentication is easy,  but most of the email servers have this fonctionnality disabled and must be enabled manually, so yes its a bit techy, or use OAuth, but implementing this is also not so easy

**Comment by tritus** [comment]
yes I'm quite bad at native dev on windows/mac. I would not be able to implement oauth in such platforms. Feel free to make a PR if you fell confortable with it.
